### ISSUE TITLE
Request support for Windows 7

### ISSUE THREAD

**Issue by amie33r** [xd003/easyclone#8]
Hello,
Is it possible for you to support for this project windows 7?

**Comment by xd003** [comment]
Yes , technically its possible if i dockerize this project then it can be used on windows 7 too .. i will add docker support when i get time as it will take some time

**Comment by amie33r** [comment]
> 
> 
> Yes , technically its possible if i dockerize this project then it can be used on windows 7 with docker .. i will add docker support when i get time as it will take some time

Actually I wanted to see whether you could compile a .exe just like rclone/gclone etc.. and I've heard that docker stopped supporting win 7 officially. But if compile an executable file is difficult, guess I will probably adjust with docker toolbox for running/copying data since gclone became crap now.

**Comment by xd003** [comment]
> > Yes , technically its possible if i dockerize this project then it can be used on windows 7 with docker .. i will add docker support when i get time as it will take some time
> 
> Actually I wanted to see whether you could compile a .exe just like rclone/gclone etc.. and I've heard that docker stopped supporting win 7 officially. But if compiling an executable file is difficult, guess I will probably adjust with docker toolbox for running/copying data since gclone became crap now.

Here's the compiled exe file - https://easyclone.xd003.workers.dev/0:/lclone/lclone.exe

**Comment by amie33r** [comment]
> 
> 
> > > Yes , technically its possible if i dockerize this project then it can be used on windows 7 with docker .. i will add docker support when i get time as it will take some time
> > 
> > 
> > Actually I wanted to see whether you could compile a .exe just like rclone/gclone etc.. and I've heard that docker stopped supporting win 7 officially. But if compiling an executable file is difficult, guess I will probably adjust with docker toolbox for running/copying data since gclone became crap now.
> 
> Here's the compiled exe file - https://easyclone.xd003.workers.dev/0:/lclone/lclone.exe

Okay, thank you, is this the same lclone from l3uddz?

**Comment by xd003** [comment]
> > > > Yes , technically its possible if i dockerize this project then it can be used on windows 7 with docker .. i will add docker support when i get time as it will take some time
> > > 
> > > 
> > > Actually I wanted to see whether you could compile a .exe just like rclone/gclone etc.. and I've heard that docker stopped supporting win 7 officially. But if compiling an executable file is difficult, guess I will probably adjust with docker toolbox for running/copying data since gclone became crap now.
> > 
> > 
> > Here's the compiled exe file - https://easyclone.xd003.workers.dev/0:/lclone/lclone.exe
> 
> Okay, thank you, is this the same lclone from l3uddz?

Yes

**Comment by amie33r** [comment]
> 
> 
> > > > > Yes , technically its possible if i dockerize this project then it can be used on windows 7 with docker .. i will add docker support when i get time as it will take some time
> > > > 
> > > > 
> > > > Actually I wanted to see whether you could compile a .exe just like rclone/gclone etc.. and I've heard that docker stopped supporting win 7 officially. But if compiling an executable file is difficult, guess I will probably adjust with docker toolbox for running/copying data since gclone became crap now.
> > > 
> > > 
> > > Here's the compiled exe file - https://easyclone.xd003.workers.dev/0:/lclone/lclone.exe
> > 
> > 
> > Okay, thank you, is this the same lclone from l3uddz?
> 
> Yes

Does this lclone have sasync in it already? I apologize if I'm asking stupid question.

**Comment by amie33r** [comment]
> 
> 
> > > > > > Yes , technically its possible if i dockerize this project then it can be used on windows 7 with docker .. i will add docker support when i get time as it will take some time
> > > > > 
> > > > > 
> > > > > Actually I wanted to see whether you could compile a .exe just like rclone/gclone etc.. and I've heard that docker stopped supporting win 7 officially. But if compiling an executable file is difficult, guess I will probably adjust with docker toolbox for running/copying data since gclone became crap now.
> > > > 
> > > > 
> > > > Here's the compiled exe file - https://easyclone.xd003.workers.dev/0:/lclone/lclone.exe
> > > 
> > > 
> > > Okay, thank you, is this the same lclone from l3uddz?
> > 
> > 
> > Yes
> 
> Does this lclone have sasync in it already? I apologize if I'm asking stupid question.

What happened? Please respond
